### PR TITLE
Update block-updates.md

### PR DESCRIPTION
**Pull request by quadsYT**

### CODE REVIEW

File: docs/extras/block-updates.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Blocking Updates
 ---
-All currently known Wii U exploits can, unlike e.g. the Nintendo Switch RCM exploit, be patched by a system update. Although the Wii U is no longer officially supported, Nintendo may still release updates for it. Namely, the updates 5.5.3 and 5.5.4 were both released after the Wii U was discontinued, so blocking updates is still a recommended action.
+All currently known Wii U exploits can, unlike e.g. the Nintendo Switch RCM exploit, be patched by a system update. Although the Wii U is no longer officially supported, Nintendo may still release updates for it. Namely, the updates 5.5.3, 5.5.4 and 5.5.5 were both released after the Wii U was discontinued, so blocking updates is still a recommended action.
```
Comment (GaryOderNichts) [inline docs/extras/block-updates.md:3]: Don't use "both" when referring to 3 updates.

**Review by GaryOderNichts — CHANGES_REQUESTED** [review]
Also the sentence is technically still valid without your changes. It's not required to list every single update to make the point that blocking updates is recommended.

File: docs/extras/block-updates.md (hunk 1 above)
Comment (quadsYT) [reply docs/extras/block-updates.md:3]: Ah okay

**Comment by quadsYT** [comment]
@GaryOderNichts  Done.

**Pull request merged**